include/zephyr/bluetooth/audio/pbp.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_PBP_
+#define ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_PBP_
+
+/**
+ * @brief Public Broadcast Profile (PBP)
+ *
+ * @defgroup bt_pbp Public Broadcast Profile (PBP)
+ *
+ * @ingroup bluetooth
+ * @{
+ *
+ * [Experimental] Users should note that the APIs can change
+ * as a part of ongoing development.
+ */
+
+#include <zephyr/sys/util.h>
+#include <zephyr/bluetooth/audio/audio.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Public Broadcast Announcement features */
+enum bt_pbp_announcement_feature {
+	/** Broadcast Streams encryption status */
+	BT_PBP_ANNOUNCEMENT_FEATURE_ENCRYPTION = BIT(0),
+	/** Standard Quality Public Broadcast Audio configuration */
+	BT_PBP_ANNOUNCEMENT_FEATURE_STANDARD_QUALITY = BIT(1),
+	/** High Quality Public Broadcast Audio configuration */
+	BT_PBP_ANNOUNCEMENT_FEATURE_HIGH_QUALITY = BIT(2),
+};
+
+/**
+ * @brief Creates a Public Broadcast Announcement based on the information received
+ * in the features parameter.
+ *
+ * @param meta		Metadata to be included in the advertising data
+ * @param features      Public Broadcast Announcement features
+ * @param meta_len	Size of the metadata fields to be included in the advertising data
```

Comment:
The parameters description order does not follow the parameters list order
```suggestion
 * @param meta_len	Size of the metadata fields to be included in the advertising data
 * @param features      Public Broadcast Announcement features
```